src/main/java/seedu/bookmark/model/book/Note.java

```diff
@@ -24,6 +24,7 @@
      * @param text A valid text string.
      */
     public Note(String title, String text) {
+        assert (title != null && text != null);
```

Comment:
is this redundant since there's a requireNonNull right after already? 😆

Reply:
This was for the assertions requirement for the week haha! Should I remove it?